src/stable/juicefs/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: juicefs
+description: A Helm chart for JuiceFS CSI Driver
+type: application
+version: 0.6.1
+appVersion: 0.10.6
+kubeVersion: ">=1.14.0-0"
+home: https://github.com/juicedata/juicefs
+sources:
+  - https://github.com/juicedata/juicefs
+keywords:
+  - juicefs
+  - csi
```

Comment:
Is it possible to add the maintainers field here? For showing it in kubesphere

Reply:
done